src/Config/Config.php

```diff
@@ -20,12 +29,22 @@ final class Config
 
     private Finder $finder;
 
-    public function __construct(string $name = 'Default')
+    private ?string $cacheFile = null;
```

Comment:
Should we always provide cache by default with a default cache file ?

Phpcsfixer use `.php-cs-fixer.cache` by default.
We could use `.twig-cs-fixer.cache`.

This way, even if people are not aware of this new feature (or doesn't want to have a custom conf) they will enjoy the caching feature.